docs/UserGuide.md

```diff
@@ -97,12 +97,12 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS g/GENDER l/SEC_LEVEL m/MRT
 
 <box type="tip" seamless>
 
-**Tip:** A student must have at least 1 tag (subject)
+**Tip:** If enrol dates are not specified, it will stick to the default (Today's date)
```

Comment:
I think we might not need to include this tip as it is already stated to the reader above: "When adding subjects, if no enrol date is provided, the date on which the command was executed will be applied to all subjects listed by default."

Reply:
Agreed.